domain/removal/state/state.go

```diff
@@ -101,15 +159,15 @@ func (st *State) RelationScheduleRemoval(
 		return errors.Capture(err)
 	}
 
-	removalRec := removal{
+	removalRec := removalJob{
 		UUID:          removalUUID,
 		RemovalTypeID: 0,
```

Comment:
`0` => `removal.RelationJob`?

Reply:
You'd have to cast it. I already use ints directly for life IDs above.